cmd/create.go

```diff
@@ -189,30 +202,25 @@ var createCmd = &cobra.Command{
 			}
 		}()
 
+		// This calls the templates
 		err = project.CreateMainFile()
-
-		// once the create is done, stop the spinner
-		spinner.Quit()
-
-		// wait for the spinner to finish
-		wg.Wait()
 		if err != nil {
 			log.Printf("Problem creating files for project. %v", err)
 			cobra.CheckErr(textinput.CreateErrorInputModel(err).Err())
 		}
 
-		fmt.Println(endingMsgStyle.Render("\nNext steps cd into the newly created project with:"))
-		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd %s\n", project.ProjectName)))
+		fmt.Println(endingMsgStyle.Render("\nNext steps:"))
+		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd into the newly created project with: `cd %s`\n", project.ProjectName)))
+
+		if options.Advanced.Choices["AddHTMXTempl"] {
+			fmt.Println(endingMsgStyle.Render("• Install the templ cli if you haven't already by running `go install github.com/a-h/templ/cmd/templ@latest`\n"))
+			fmt.Println(endingMsgStyle.Render("• Generate templ function files by running `templ generate`\n"))
+		}
 
 		if isInteractive {
 			nonInteractiveCommand := utils.NonInteractiveCommand(cmd.Use, cmd.Flags())
 			fmt.Println(tipMsgStyle.Render("Tip: Repeat the equivalent Blueprint with the following non-interactive command:"))
 			fmt.Println(tipMsgStyle.Italic(false).Render(fmt.Sprintf("• %s\n", nonInteractiveCommand)))
-			err = tprogram.ReleaseTerminal()
-			if err != nil {
-				log.Printf("Could not release terminal: %v", err)
-				cobra.CheckErr(err)
-			}
```

Comment:
I think this is potentially causing an issue with the current branch. I cannot exit out of the process. See screenshot.
<img width="952" alt="Screenshot 2023-12-18 at 4 30 44 PM" src="https://github.com/Melkeydev/go-blueprint/assets/53410236/2b6ad1c1-4ae5-44d7-8e96-3619259cf5fd">


We want that if a user kills the process in the terminal, it stops. This is seemingly hanging.

Reply:
Fixed; Solving new bug is in process, but this one was not related to terminal release. I saw it earlier, but I didn't pay attention. I thought it was problem in my terminal.